packages/vite-plugin-cloudflare/e2e/basic.test.ts

```diff
@@ -3,13 +3,27 @@ import { fetchJson, runCommand, test, waitForReady } from "./helpers.js";
 
 describe("node compatibility", () => {
 	describe.each(["pnpm --no-store", "npm", "yarn"])("using %s", (pm) => {
-		test("can serve a Worker request", async ({ expect, seed, viteDev }) => {
+		test("can serve a Worker request", async ({
+			expect,
+			seed,
+			viteDev,
+			vitePreview,
+		}) => {
 			const projectPath = await seed("basic");
 			runCommand(`${pm} install`, projectPath);
 
-			const proc = await viteDev(projectPath);
-			const url = await waitForReady(proc);
-			expect(await fetchJson(url + "/api/")).toEqual({ name: "Cloudflare" });
+			const devProc = await viteDev(projectPath);
+			const devUrl = await waitForReady(devProc);
+			expect(await fetchJson(devUrl + "/api/")).toEqual({ name: "Cloudflare" });
+
+			const command = pm === "npm" ? `npx` : pm;
+			runCommand(`${command} vite build`, projectPath);
```

Comment:
Would be nice to wrap this in a `viteBuild` function so we have one for dev, build and preview.